cpp/dolfinx/geometry/utils.h

```diff
@@ -663,40 +663,37 @@ graph::AdjacencyList<std::int32_t> compute_colliding_cells(
 /// @param[in] mesh The mesh
 /// @param[in] points Points to check for collision (`shape=(num_points,
 /// 3)`). Storage is row-major.
+/// @param[in] cells Cells to check for ownership
 /// @param[in] padding Amount of absolute padding of bounding boxes of the mesh.
 /// Each bounding box of the mesh is padded with this amount, to increase
 /// the number of candidates, avoiding rounding errors in determining the owner
 /// of a point if the point is on the surface of a cell in the mesh.
-/// @return Tuple `(src_owner, dest_owner, dest_points, dest_cells)`,
+/// @return Point ownership data with fields src_owner, dest_owner, dest_points, dest_cells,
 /// where src_owner is a list of ranks corresponding to the input
 /// points. dest_owner is a list of ranks corresponding to dest_points,
 /// the points that this process owns. dest_cells contains the
 /// corresponding cell for each entry in dest_points.
 ///
 /// @note `dest_owner` is sorted
-/// @note Returns -1 if no colliding process is found
+/// @note `src_owner` is -1 if no colliding process is found
 /// @note dest_points is flattened row-major, shape `(dest_owner.size(),
 /// 3)`
-/// @note Only looks through cells owned by the process
+/// @note Only returns cells owned by the process
```

Comment:
Do you by this mean that `dest_cells` are only local (which they are).
I think the note can be removed, as in theory, you could now send in any set of cells (owned or ghosted) in the `cells` array.

Reply:
Yes that is what was meant. I will remove the note.